forecastSrc/bbc_forecast_data.py

```diff
@@ -1,144 +1,167 @@
 #!/usr/bin/python2.7
 
+
 # -*- coding: utf-8 -*-
 def bbc_forecast_data():
-    #Using weather forecast provided by BBC.
-    import urllib
+    # Using weather forecast provided by BBC.
     import datetime
+    import urllib
     from xml.dom import minidom
-    from forecast_db_interface import forecast_db_interface, BbcTable, toFloat
 
-    #url = 'http://tinyurl.com/bbc3dayforecast'
-    url = 'http://open.live.bbc.co.uk/weather/feeds/en/3133880/3dayforecast.rss'
+    from forecast_db_interface import BbcTable, forecast_db_interface, toFloat
+
+    # url = 'http://tinyurl.com/bbc3dayforecast'
+    url = "http://open.live.bbc.co.uk/weather/feeds/en/3133880/3dayforecast.rss"
 
     dom = minidom.parse(urllib.urlopen(url))
-    forecast = dom.getElementsByTagName('channel')[0]
+    forecast = dom.getElementsByTagName("channel")[0]
 
     dbIf = forecast_db_interface()
     dbIf.createTables()
 
     raw_forecasts = []
     dated_forecast = {}
 
-
-    daysOfWeek   = ['Mon','Tue','Wed','Thu','Fri','Sat','Sun']
-    monthsOfYear = ['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec']
-
-    startDateText           = forecast.getElementsByTagName('pubDate')[0].toxml()[9:25]
-    startDay,startDateText  = startDateText.split(',')
-    startDateText           = startDateText.lstrip()
-    dy,mon,yr               = startDateText.split(' ')
+    daysOfWeek = ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun"]
+    monthsOfYear = [
+        "Jan",
+        "Feb",
+        "Mar",
+        "Apr",
+        "May",
+        "Jun",
+        "Jul",
+        "Aug",
+        "Sep",
+        "Oct",
+        "Nov",
+        "Dec",
+    ]
+
+    startDateText = forecast.getElementsByTagName("pubDate")[0].toxml()[9:25]
+    startDay, startDateText = startDateText.split(",")
+    startDateText = startDateText.lstrip()
+    dy, mon, yr = startDateText.split(" ")
 
     dates = []
-    dates.append(datetime.date(int(yr),monthsOfYear.index(mon[0:3])+1,int(dy)))
+    dates.append(datetime.date(int(yr), monthsOfYear.index(mon[0:3]) + 1, int(dy)))
     dates.append(dates[0] + datetime.timedelta(days=1))
     dates.append(dates[1] + datetime.timedelta(days=1))
 
-    for node in forecast.getElementsByTagName('item'):
-        title       = node.getElementsByTagName('title')[0]
```

Comment:
Same here. I would prefer to have the code aligned so that all the "=" symbols are in one column. Makes for better code readability in my opinion. Could you please revert this?